src/grammar-builder/grammar.ts

```diff
@@ -178,7 +178,15 @@ export function alias (alias : string, rule : Rule) : AliasRule {
     };
 }
 
+export function precedence (prec : number, rule : RuleObj) : RuleObj {
+    return {
+        ...rule,
+        precedence : prec,
```

Comment:
Lets us set the precedence for any rule.

Precedence only has an effect if we do not inline the rule, however...